tests/test_asn1.ml

```diff
@@ -70,11 +79,18 @@ module Rsa = struct
     [ "Private" >:: test_priv ~decode expected_private (fixture "rsa_pkcs8.der")
     ]
 
+  let test_negative_key =
+    let decode = Key_parsers.Asn1.X509.decode_rsa in
+    "X509" >:::
+    [ "Negative Public Key" >:: test_pub ~decode expected_negative (fixture "negative_rsa.der")
```

Comment:
I had a look at this file and it looks like its `n` is positive.  If that's indeed the case, then the test won't go through the new fix.

Reply:
Correct. I though OpenSSL was being too smart, but it wasn't. Fixed it.